translator/trace/jaeger/protospan_to_jaegerproto.go

```diff
@@ -134,6 +135,26 @@ func ocNodeToJaegerProcessProto(node *commonpb.Node) *jaeger.Process {
 		serviceName = node.ServiceInfo.Name
 	}
 
+	if resource != nil {
+		resourceType := resource.GetType()
+		if resourceType != "" {
+			jTags = append(jTags, jaeger.KeyValue{
+				Key:   opencensusResourceType,
+				VType: jaeger.ValueType_STRING,
+				VStr:  resourceType,
+			})
+		}
+		for k, v := range resource.GetLabels() {
+			if v != "" {
```

Comment:
Why are we ignoring empty values?

Reply:
I don't have a strong opinion on this. I suggest ignoring them because for OTel resources not all labels are required (e.g pod_id is optional in gke_container resource). For labels without a value I don't think they're pretty useful.

Reply:
I don't have a strong opinion either but I think it's probably better to allow it. I don't see a reason why anyone would use an empty string as a value but I don't see much reason in disallowing it either. If something does not have any obvious downsides, I think we should generally try to not limit flexibility for end users. The check for resource type is a different thing as empty string there means it was never set.

Reply:
@owais so do you plan to remove this `if`?